swarm/storage/ldbstore.go

```diff
@@ -290,8 +320,8 @@ func (s *LDBStore) collectGarbage(ratio float32) {
 
 	sort.Slice(garbage[:gcnt], func(i, j int) bool { return garbage[i].value < garbage[j].value })
```

Comment:
why is this sorting needed, was it not the whole point not to have to sort

Reply:
Yes it's superflouous now. Thanks.